lib/kennel.rb

```diff
@@ -58,12 +61,19 @@ def generate
     end
 
     def plan
-      syncer.plan
+      the_plan = syncer.plan
+      PlanResults.new(
+        plan: the_plan
+      )
     end
 
     def update
-      syncer.plan
-      syncer.update if syncer.confirm
+      the_plan = syncer.plan
+      the_update = syncer.update if syncer.confirm
+      UpdateResults.new(
```

Comment:
singular since it is 1 result ?
```suggestion
      UpdateResult.new(
```

Reply:
Sure. This name is internal to the kennel engine anyway.